pkg/modulereader/hcl_utils.go

```diff
@@ -36,10 +36,11 @@ func getHCLInfo(source string) (ModuleInfo, error) {
 	ret := ModuleInfo{}
 
 	if sourcereader.IsEmbeddedPath(source) {
-		if !tfconfig.IsModuleDirOnFilesystem(tfconfig.WrapFS(ModuleFS), source) {
+		wrapFS := tfconfig.WrapFS(sourcereader.ModuleFS)
```

Comment:
It appears to me that this has been refactored to avoid introducing the problem from https://github.com/hashicorp/terraform-config-inspect/issues/68. If action is required, please take it.

Reply:
That doesn't seem to be an issue `WrapFS` does solve this issue.